docs/content/usage/concepts.md

```diff
@@ -30,9 +30,20 @@ This allows you to leverage the latest and greatest features of the AVM modules,
 
 A private registry - that is hosted in your own environment - can store modules originating from the public registry. Using a private registry still grants you the latest version of AVM modules while allowing you to review each version of each module before admitting them to your private registry. You also have control over who can access your own private registry. Note that using a private registry means that you're still using each module as is, without making any changes.
 
-#### Inner-sourcing (using a Private Repository)
+#### Inner-sourcing (using your own private registry, repositories or other storage options)
```

Comment:
While I see why you'd want to mention it in the title, I would not include a private registry in this title and would keep the focus on where the source code is kept (private repository); if you want to mention it, at least try not leading with it in the title. 

Note: the private registry scenario is mentioned above.